connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -95,6 +97,121 @@ class Column private[sql] (private[sql] val expr: proto.Expression) extends Logg
   def name(alias: String): Column = Column { builder =>
     builder.getAliasBuilder.addName(alias).setExpr(expr)
   }
+
+  /**
+   * Returns a sort expression based on the descending order of the column.
+   * {{{
+   *   // Scala
+   *   df.sort(df("age").desc)
+   *
+   *   // Java
+   *   df.sort(df.col("age").desc());
+   * }}}
+   *
+   * @group expr_ops
+   * @since 1.3.0
```

Comment:
(should probably fix these version info)

Reply:
yeah will do